plugin/build.gradle

```diff
@@ -52,6 +52,7 @@ dependencies {
     api 'androidx.core:core-ktx:1.0.1'
     api 'com.google.android.material:material:1.1.0-alpha05'
     api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+    api "com.takisoft.preferencex:preferencex-simplemenu:$preferencexVersion"
```

Comment:
Why?

Reply:
PreferenceThemeOverlay depend preference

Reply:
In that case, moving the fix to core is preferred.